### PR TITLE
chore(doc-spa): Updating SPA deployment guide

### PR DESCRIPTION
**Pull request by ayushsatyam146**
# Closes/Fixes/Resolves
ONEPLAT-2485
### 

# Explain the feature/fix
Updating SPA deployment guide to point to the SPAship quickstart deployment guide.

## Does this PR introduce a breaking change
No

### CODE REVIEW

**Review by deshmukhmayur — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

File: packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md [hunk 1]
```diff
@@ -1,33 +1,10 @@
 ---
 id: spa-deployment-guideline
-title: OP SPA Deployment Guideline
+title: SPA Deployment Guideline
```
Comment (riginoommen) [inline packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md:3]:
```suggestion
title: SPA Deployment Guidelines
```

File: packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md [hunk 2]
```diff
@@ -1,33 +1,10 @@
 ---
 id: spa-deployment-guideline
```
Comment (riginoommen) [inline packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md:2]:
```suggestion
id: spa-deployment-guidelines
```

**Review by riginoommen — COMMENTED** [review]
Update the parent which references this document

**Review by riginoommen — CHANGES_REQUESTED** [review]
Please see comments

File: packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md [hunk 3]
```diff
@@ -1,33 +1,10 @@
 ---
-id: spa-deployment-guideline
-title: OP SPA Deployment Guideline
+id: spa-deployment-guidelines
+title: SPA Deployment Guidelines
 slug: /deployments/spa
 sidebar_label: SPA Guideline
```
Comment (riginoommen) [inline packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md:5]:
```suggestion
sidebar_label: SPA Guidelines
```

File: packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md [hunk 4]
```diff
@@ -1,33 +1,10 @@
 ---
-id: spa-deployment-guideline
-title: OP SPA Deployment Guideline
+id: spa-deployment-guidelines
+title: SPA Deployment Guidelines
 slug: /deployments/spa
```
Comment (riginoommen) [inline packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md:4]: Update slug also

**Review by riginoommen — APPROVED** [review]

**Pull request merged**